docs/introduction-to-smart-contracts.rst

```diff
@@ -366,6 +366,14 @@ accessing (either reading or writing) a previously untouched memory word (ie. an
 within a word). At the time of expansion, the cost in gas must be paid. Memory is more
 costly the larger it grows (it scales quadratically).
 
+Also, if you want to create a new struct variable inside a function, 
```

Comment:
This place in the documentation is way too early. It talks about the EVM, structs as a concept are not introduced yet. Perhaps it is better to add that to the structs, arrays or mapping section, or introduce some new section after those.

Reply:
Sure will do that

Reply:
@chriseth sent in https://github.com/ethereum/solidity/pull/3977